juntagrico/templates/fragments/subscription_selection.html

```diff
@@ -0,0 +1,62 @@
+{% load i18n %}
```

Comment:
nicely done 
one little single naming convention thingy
such teplate filles are called snippet_blablabla
ther are already two of them
so i would suggest call the folder snippets and move all of them there and rename this file